migrations/m3_acp_module.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ *
+ * Ideas extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2015 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\ideas\migrations;
+
+class m3_acp_module extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return array('\phpbb\ideas\migrations\m2_initial_data');
```

Comment:
Should depend on m1. Technically this does not need to depend on anything, so in that case we at least depend on the first migration of this ext.